### PR TITLE
Addition of VS code reference

### PR DESCRIPTION
**Pull request by EldaveO**
:new: :bomb: :mortar_board: :memo: :robot: :lipstick: :bug: :telescope: :art: Add relevant emojis to title. [See style guide.](https://github.com/jameshughes89/cs101/blob/main/CONTRIBUTING.md#style-guidelines).

### Related Issues or PRs

If applicable, replace this text with links to issues and other PRs. [Link with a keyword if you can.](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue)

### What

Addition of VS code as an option in the website documentation

### Why

I found VScode to be pretty friendly to use as a beginner for a python environment. It was also easily to implement on a Raspberry Pi. I felt that the students may benefit in the same way I did when using it.
Additional Python packages were quite easy to import as well.

### Where

\site\topics\values-variables-types 
near the end of the section.

### How

Addition of a a single line   
* VS Code (Windows, Linux, RaspberryPi)

### Testing

If applicable, replace this text with a detailed description of how changes were tested. Include manual testing.

### Additional Notes

If applicable, replace this text with additional notes. Copyright concerns? etc.

### CODE REVIEW

File: site/topics/values-variables-types/values-variables-types.rst [hunk 1]
```diff
@@ -382,6 +382,7 @@ Doing sequences of things
    * Notepad++ (Windows)
    * Sublime (Windows and Mac)
    * Integrated Development Environment
+   * VS Code (Windows, Linux, RaspberryPi)
```
Comment (twentylemon) [inline site/topics/values-variables-types/values-variables-types.rst:385]: Why is pi included here?

Comment (EldaveO) [reply site/topics/values-variables-types/values-variables-types.rst:385]: I should possibly say Raspberrian OS, but I felt that students who have heard of the Raspberry Pi but not looked into it too deeply may get inspired to look at it as a low powered computing solution.

Comment (twentylemon) [reply site/topics/values-variables-types/values-variables-types.rst:385]: How is that related the course content?

Comment (EldaveO) [reply site/topics/values-variables-types/values-variables-types.rst:385]: It's providing a low barrier introductory pathway for 1st year students to learn to develop on a system other than Mac or Windows.

Comment (twentylemon) [reply site/topics/values-variables-types/values-variables-types.rst:385]: To be clear, vscode is reasonable to add. It's more lightweight than intellij. But pi is irrelevant, it's not an operating system people are likely to be using.

Comment (EldaveO) [reply site/topics/values-variables-types/values-variables-types.rst:385]: It's not that they're using it right now, but in the past maybe in high school or younger they may have used it and put it down because it was "Too Hard" or "frustrating" when compared to a more modern OS. 

The comment may give them a reason to pick it up again and develop on the Pi platform again. Maybe buy one to act as a Minecraft server, or learn how to SSH into a device. 

The pi comment was meant to attempt to spark curiosity and encourage self directed learning.

Comment (twentylemon) [reply site/topics/values-variables-types/values-variables-types.rst:385]: All right, I'm going to write way too much, here goes.

The course is fundamentals of computer science, basic data structures, algorithms, all that good stuff. That's (hopefully) achieved through theory (via lectures) and practice (via programming). Python is the tool of practice here to help keep the abstraction level high -- in a course about computer science, we don't really want to get bogged down by _computers_.

Ok. So what is this section then? Writing python requires editing text files at the end of the day, this is meant to point students in the right direction to get off the ground in order to be writing their python code. Programs are platform dependent however, so the platform is included in descriptions here. The three mentioned (windows, mac, linux) will cover every student. The pi is not an operating system I'd expect any student to be using, let alone wanting to use for the course (plus, it's linux anyways).

Not to say the points you raise are wrong or even bad, rather to say they are unrelated to the course objectives, and in my opinion, will serve as noise. So in the spirit of _I don't know, I don't wanna know_, I'd say cut the mention of the pi.

`code` is a good IDE, it deserves a mention here. The pi does not.

Comment (jameshughes89) [reply site/topics/values-variables-types/values-variables-types.rst:385]: > Not to say the points you raise are wrong or even bad, rather to say they are unrelated to the course objectives, and in my opinion, will serve as noise. So in the spirit of I don't know, I don't wanna know, I'd say cut the mention of the pi.

This. 

It is defo isn't a bad point at all --- I can see Pi serving as a way to spark interest, but at this stage I too feel that it may be noise. Also, I fear it could add to confusion to students unfamiliar with CS/computers. It may also open up multiple can of worms:
- Operating Systems? Pi is an OS? Why PiOS? That's linux too? What? (╯°□°)╯︵ ┻━┻
- Processors/instruction sets? What is a bytecode  (╯°□°)╯︵ ┻━┻

All important ideas, but something that will intimidate someone who just wants to have their computer print `Hello, World`. 

**Long story short** Replace Pi with Mac, and we'll send it.

Comment (jameshughes89) [reply site/topics/values-variables-types/values-variables-types.rst:385]: Did you push?

Comment (EldaveO) [reply site/topics/values-variables-types/values-variables-types.rst:385]: I think so, Its updated on my fork. It seems like it didn't push to here.
